extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -42,6 +42,9 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverAudienceNetworkBidderCountries';
 		$vars[] = 'wgAdDriverA9BidderCountries';
 		$vars[] = 'wgAdDriverA9VideoBidderCountries';
+		$vars[] = 'wgAdDriverBabDetectionDesktopCountries';
+		$vars[] = 'wgAdDriverBabDetectionMobileCountries';
+		$vars[] = 'wgAdDriverBabDetectionFandomCountries';
```

Comment:
This one should go here:
https://github.com/Wikia/app/pull/14554/files#diff-ac1bcc3ae7cf3e1fa4ec596d457b1bdcR110
Please keep F2 naming convention: `wgAdDriverF2InstartLogicRecoveryCountries`

Do we want to keep separate variables for all platforms? Why don't we use the same for all?

Reply:
Renamed and moved. We use separate variables because we want to turn this on gradually. Also discussed with Damian and Nef.